### PR TITLE
Audit all docstrings for style, typos and outdated info

### PR DESCRIPTION
**Pull request by systemcatch**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪  Added **tests** for changed code.
- [ ] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [ ] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚  Updated **documentation** for changed code.
- [ ] ⏩  Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

## Description of Changes

Closes: https://github.com/arrow-py/arrow/issues/819
Closes: #946

### CODE REVIEW

**Comment by jadchaar** [comment]
Is this ready for review @systemcatch?

**Comment by systemcatch** [comment]
Ready for a first review!

File: arrow/arrow.py [hunk 1]
```diff
@@ -305,13 +306,6 @@ def fromdatetime(cls, dt: dt_datetime, tzinfo: Optional[TZ_EXPR] = None) -> "Arr
         :param tzinfo: (optional) A :ref:`timezone expression <tz-expr>`.  Defaults to ``dt``'s
             timezone, or UTC if naive.
 
-        If you only want to replace the timezone of naive datetimes::
-
-            >>> dt
-            datetime.datetime(2013, 5, 5, 0, 0, tzinfo=tzutc())
-            >>> arrow.Arrow.fromdatetime(dt, dt.tzinfo or 'US/Pacific')
-            <Arrow [2013-05-05T00:00:00+00:00]>
```
Comment (jadchaar) [inline arrow/arrow.py:313]: Should we replace this with a better example so we have at least one example in the docstring? Or for these "datetime" compatibility functions, do you think it is unnecessary?

File: arrow/arrow.py [hunk 2]
```diff
@@ -849,16 +846,16 @@ def timestamp(self) -> float:
...
+            1616882340.256501
 
         """
 
         return self._datetime.timestamp()
 
     @property
     def int_timestamp(self) -> int:
-        """Returns a timestamp representation of the :class:`Arrow <arrow.arrow.Arrow>` object, in
+        """Returns a integer timestamp representation of the :class:`Arrow <arrow.arrow.Arrow>` object, in
```
Comment (jadchaar) [inline arrow/arrow.py:858]: small grammar: "Returns an integer"

File: arrow/arrow.py [hunk 3]
```diff
@@ -1653,7 +1652,11 @@ def _get_datetime(
 
     @classmethod
     def _get_frames(cls, name: _T_FRAMES) -> Tuple[str, str, int]:
+        """Finds relevant timeframe and steps for use in range and span methods.
+
+        Returns a 3 element tuple in the form ("day", "days", 1.)
```
Comment (jadchaar) [inline arrow/arrow.py:1657]: Should we add the word example here so it is obvious this is an example output tuple and not the only one returned?

File: arrow/constants.py [hunk 4]
```diff
@@ -1,3 +1,8 @@
+"""
+Constants used internally in arrow..
+
+"""
```
Comment (jadchaar) [inline arrow/constants.py:4]: This can probably made into a one line docstring.

Also you got a double period.

File: arrow/formatter.py [hunk 5]
```diff
@@ -1,3 +1,8 @@
+"""
+Provides the :class:`Arrow <arrow.formatter.DateTimeFormatter>` class, an improved formatter for datetimes.
+
+"""
```
Comment (jadchaar) [inline arrow/formatter.py:4]: Should these top of file comments be made into one line docstrings?

File: arrow/util.py [hunk 6]
```diff
@@ -57,7 +62,11 @@ def is_timestamp(value: Any) -> bool:
 
 
 def validate_ordinal(value: Any) -> None:
-    """Raise the corresponding exception if value is an invalid Gregorian ordinal."""
+    """Raise an  exception if value is an invalid Gregorian ordinal.
```
Comment (jadchaar) [inline arrow/util.py:65]: Two spaces

**Review by jadchaar — CHANGES_REQUESTED** [review]
Looks good, but a few tweaks.

File: arrow/arrow.py (hunk 1 above)
Comment (systemcatch) [reply arrow/arrow.py:313]: I can replace it with a better example.

File: arrow/arrow.py [hunk 7]
```diff
@@ -305,12 +306,12 @@ def fromdatetime(cls, dt: dt_datetime, tzinfo: Optional[TZ_EXPR] = None) -> "Arr
...
             timezone, or UTC if naive.
 
-        If you only want to replace the timezone of naive datetimes::
+        Usage::
 
             >>> dt
-            datetime.datetime(2013, 5, 5, 0, 0, tzinfo=tzutc())
-            >>> arrow.Arrow.fromdatetime(dt, dt.tzinfo or 'US/Pacific')
-            <Arrow [2013-05-05T00:00:00+00:00]>
+            datetime.datetime(2021, 4, 7, 13, 48)
```
Comment (jadchaar) [inline arrow/arrow.py:312]: Should we make this an tz-aware datetime? Works both ways, just wanted to ask.

File: arrow/locales.py [hunk 8]
```diff
@@ -1,3 +1,5 @@
+"""Provides internationalization for arrow in over 60 languages and dialects."""
```
Comment (jadchaar) [inline arrow/locales.py:1]: Love the addition of the number of locales 🙌!

File: arrow/util.py [hunk 9]
```diff
@@ -1,3 +1,8 @@
+"""
+Helpful functions used internally within arrow.
+
+"""
```
Comment (jadchaar) [inline arrow/util.py:4]: Single line

**Review by jadchaar — APPROVED** [review]
Small tweaks, but looks great! This is a good first step. Next step (in a new PR) is to probably integrate a docstring style checker to make sure all of them are consistent, but this is a great start :D!

**Review by krisfremen — APPROVED** [review]
Nice.

I had a few comments in a draft review, seems like @jadchaar covered all of them, looks good to me.

File: arrow/arrow.py (hunk 7 above)
Comment (systemcatch) [reply arrow/arrow.py:312]: Can do!

**Pull request merged**